idunn/utils/categories.yml

```diff
@@ -1,144 +1,602 @@
+          - "supermarchés, hypermarchés"
         raw_filters:
-            - "*,supermarket"
-            - "*,mall"
+          - "*,supermarket"
+          - "*,mall"
         regex: "superette|epicier|epicerie|marche|market"
-    bank:
-        pj_what: "banques"
+
+    education:
+        pj_what: "école collège lycée"
+        pj_filters:
+          - "écoles maternelles publiques"
+          - "écoles primaires publiques"
+          - "collèges publics"
+          - "lycées d'enseignement général et technologique publics"
+          - "écoles maternelles privées"
+          - "écoles primaires privées"
+          - "collèges privés"
+          - "lycées d'enseignement général et technologique privés"
+          - "enseignement supérieur public"
+          - "enseignement supérieur privé"
+        raw_filters:
+          - "school,*"
+
+    # NOTE: This is a legacy alias for `post_office`
+    service:
+        pj_what: "poste"
+        pj_filters:
+          - "envoi, distribution de courrier, de colis"
+        raw_filters:
+          - "post_office,*"
+        regex: "(^| )post"
+
+    # NOTE: This is a legacy alias for `sport_center`
+    fitness:
+        pj_what: "sport"
+        pj_filters:
+          - "infrastructures de sports et loisirs"
+          - "piscines (établissements)"
+          - "clubs de sport"
+        raw_filters:
+          - "sports_centre,*"
+        regex: "fitness|muscu|piscine"
+
+    food_french:
+        raw_filters:
+          - "restaurant,french"
+        pj_what: "restaurant français"
+
+    food_pizza:
+        raw_filters:
+          - "restaurant,pizza"
+        pj_what: "pizzaria"
+
+    food_burger:
+        raw_filters:
+          - "restaurant,burger"
+        pj_what: "burger"
+
+    food_italian:
+        raw_filters:
+          - "restaurant,italian"
+        pj_what: "restaurant italien"
+
+    food_kebab:
+        raw_filters:
+          - "restaurant,kebab"
+        pj_what: "kebab"
+
+    food_sandwich:
+        raw_filters:
+          - "restaurant,*sandwich*"
+        pj_what: "sandwicherie"
+
+    food_asian:
+        raw_filters:
+          - "restaurant,*asian*"
+        pj_what: "restaurant asiatique"
+
+    food_japanese:
+        raw_filters:
+          - "restaurant,*japanese*"
+        pj_what: "restaurant japonais"
+
+    food_chinese:
+        raw_filters:
+          - "restaurant,*chinese*"
+        pj_what: "restaurant chinois"
+
+    food_crepe:
+        raw_filters:
+          - "restaurant,*crepe*"
+        pj_what: "crêperie"
+
+    food_indian:
+        raw_filters:
+          - "restaurant,*indian*"
+        pj_what: "restaurant indien"
+
+    food_thai:
+        raw_filters:
+          - "restaurant,*thai*"
+        pj_what: "restaurant thaï"
+
+    food_vietnamese:
+        raw_filters:
+          - "restaurant,*vietnamese*"
+        pj_what: "restaurant vietnamien"
+
+    food_lebanese:
+        raw_filters:
+          - "restaurant,*lebanese*"
```

Comment:
It could help to implement this kind of behavior, for example in cases where subclass would contain several comma-separated values (or maybe imposm actually split those?), but it is quite irrelevant now and is mostly an omission in the translation of previously existing raw SQL filters :/